### PR TITLE
actool: patch-manifest: add dependencies

### PR DESCRIPTION
**Pull request by alban**
Example:
```
$ actool patch-manifest --replace \
        --dependencies=example.com/image-a:example.com/image-b \
        ./etcd-v2.2.2-linux-amd64.aci
```

I would like to have this flag `--dependencies` for adding tests in rkt, related to https://github.com/coreos/rkt/issues/1752.

### CODE REVIEW

File: actool/manifest.go [hunk 1]
```diff
@@ -245,6 +248,20 @@ func patchManifest(im *schema.ImageManifest) error {
 			app.Isolators = append(app.Isolators, *isolator)
 		}
 	}
+
+	if patchDependencies != "" {
+		dependencies := strings.Split(patchDependencies, ":")
```
Comment (iaguis) [inline actool/manifest.go]: what about versions? they use ":" as a separator

Comment (iaguis) [inline actool/manifest.go]: I see that the version is encoded as a label in Dependency field, we could somehow support that

**Comment by alban** [comment]
In the end, I wrote a test without `--dependencies=...`. See https://github.com/coreos/rkt/pull/1820. I don't know if this PR is useful. Thoughts?

**Comment by jonboulle** [comment]
I'd rather avoid adding new functionality to actool, in favour of moving to acbuild.

**Comment by alban** [comment]
Ok, closing since I didn't use this in the end.

**Pull request closed without merging**